Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -682,6 +713,52 @@ namespace O3DE::ProjectManager
         
     }
 
+    void ProjectsScreen::HandleOpenProjectExportSettings(const QString& projectPath)
+    {
+        AZ::Outcome<EngineInfo> engineInfoResult = PythonBindingsInterface::Get()->GetProjectEngine(projectPath);
+        AZ::Outcome projectBuildPathResult = ProjectUtils::GetProjectBuildPath(projectPath);
+
+        auto engineInfo = engineInfoResult.TakeValue();
+        auto buildPath = projectBuildPathResult.TakeValue();
+
+        QString projectName = tr("Project");
+        auto getProjectResult = PythonBindingsInterface::Get()->GetProject(projectPath);
+        if (getProjectResult)
+        {
+            projectName = getProjectResult.GetValue().m_displayName;
+        }
+        else
+        {
+            QMessageBox::warning(this, tr("Tool Error"), tr("Failed to retrieve project information."), QMessageBox::Ok);
```

Comment:
This is a warning? Looks like an error, since you cannot proceed with the export. Also, you may want some detail about why it couldn't get the profile information. (If there is no error string in the result, at least provide the path to the project)